nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/WebServer.java

```diff
@@ -254,6 +255,11 @@ public Builder config(Config config) {
             config.get("content-encoding")
                     .as(ContentEncodingContext::create)
                     .ifPresent(this::contentEncodingContext);
+            // Configure media support
+            config.get("media-support")
+                    .as(MediaContext::create)
+                    // MediaContext always needs to be refreshed after config change
+                    .ifPresentOrElse(this::mediaContext, () -> mediaContext(MediaContext.create()));
```

Comment:
this should not use `ifPresentOrElse`, just `ifPresent`.
If there is no media-support in config, it should use whatever is configured by user explicitly. The current approach would always replace mediacontext with a default one, even if already configured.
So set it to `null` by default, and then in `build` method check if null, and if so, use default `MediaContext.create()`

Reply:
done